airflow/timetables/interval.py

```diff
@@ -148,7 +148,7 @@ def __init__(self, cron: str, timezone: Union[str, Timezone]) -> None:
         self.description = interval_description
 
     @classmethod
-    def deserialize(cls, data: Dict[str, Any]) -> "Timetable":
+    def deserialize(cls, data: Dict[str, Any]) -> "_CronMixin":
```

Comment:
This should still return `Timetable` since it should only be called against a timetable subclass

Reply:
Agree. Fixed by https://github.com/apache/airflow/pull/23662/commits/babb8ba9d122d69a3ec6dce4c1c7e28c8ba0ed6e